core/workspace_svg.js

```diff
@@ -126,7 +126,7 @@ Blockly.WorkspaceSvg = function(options,
         Blockly.Procedures.flyoutCategory);
   }
 
-  Blockly.blockRendering.init();
+  Blockly.blockRendering.init(this.options.renderer || 'geras');
```

Comment:
Should this happen in inject instead since the rendering is shared by the workspace, the toolbox flyout, and the trash flyout and this will cause it to be configured three times? Alternately, init could return a renderer that gets set on the workspace_svg. That would allow rendering blocks in the toolbox  differently than on the workspace which some developers do...

Reply:
Discussed offline, will tackle this in a separate PR, but agreed to move away from a global singleton rendering instance.